src/gimelstudio/core/node/property.py

```diff
@@ -144,6 +145,29 @@ def CreateUI(self, parent, sizer):
         self.AddToFoldPanel(sizer, fold_panel, self.img)
 
 
+class ColorProp(Property):
+    """ 
+    Allows the user to select a color.
+    """
+    def __init__(self, idname, default=(255, 255, 255), visible=True):
+        Property.__init__(self, idname, default, fpb_label, expanded, visible)
+    
+    def CreateUI(self, parent, sizer):
+        fold_panel = self.CreateFoldPanel(sizer)
+        fold_panel.SetBackgroundColour(wx.Colour("#464646"))
+
+        self.colorPicker = ColorPickerButton(fold_panel,
+                                        default_value=self.default,
```

Comment:
The colorpicker widget accepts a parameter called ``default``. 

It's also better to use the ``GetValue`` method to get the latest value (that way we know that the widget will have the most recent value).

``default=self.GetValue()``